execution_workflows/QAPA/README.md

```diff
@@ -0,0 +1,11 @@
+# APAeval nextflow pilot run
```

Comment:
Can you try to run this on Nextflow Tower with the files inside an S3 bucket? I'll send you the credentials